app/src/lib/stores/helpers/branch-pruner.ts

```diff
@@ -225,7 +238,7 @@ export class BranchPruner {
 
         if (isDeleted) {
           log.info(
-            `[BranchPruner] Pruned branch ${branchName} (was ${branch.sha})`
```

Comment:
We logged the explicit `sha` value here so that the user could find this value in the logs if they wished to restore this ref manually. Can we add that back in?

Reply:
ah, i was missing that context for why it was there. will get that back! :)

Reply:
fe05dd9 and 2b79154 should have us covered :)